src/index.jsx

```diff
@@ -13,21 +13,32 @@ import { registerCoreBlocks } from '@wordpress/block-library';
  * Internal dependencies
  */
 import { initializeApiFetch } from './utils/api-fetch-setup';
-import { getGBKit, getPost } from './utils/bridge';
+import { getGBKit, getPost, waitForGBKit, editorLoaded } from './utils/bridge';
 import Layout from './components/layout';
 import './index.scss';
+import EditorLoadError from './components/editor-load-error';
 
-window.GBKit = getGBKit();
-initializeApiFetch();
-initializeEditor();
+try {
+	await waitForGBKit();
```

Comment:
The web app now awaits for the presence of the `window.GBKit` global, as Android does not have a lifecycle hook that guarantees the globals will be present before the editor JavaScript runs.